### PR TITLE
feature: add preview in cms pages collection

### PR DESCRIPTION
**Pull request by ben196888**
# Description

Add pages preview in pages collection. The preview includes following basic components.
- Banner
- Headline
- Section
- TwoColumns
- ThreeColumns

## Major

- Add Pages collection preview in CMS admin page

# Impaction

## Screenshots

Please attach screenshots of UI accordingly. If you don't have any UI changes, please remove this section.

| Scenarios  | Before | After |
| ---------- | ------ | ----- |
| Admin page / Pages collection | Screenshot 2566-08-27 at 16 52 19 | Screenshot 2566-08-27 at 16 52 30 Screenshot 2566-08-27 at 16 52 37 Screenshot 2566-08-27 at 16 52 39 |

### Build and deploy time

| item       | Before | After  |
| ---------- | ------ | ------ |
| w/ Cached  | 02m06s | 02m14s |

### LightHouse

N/A

### CODE REVIEW

File: homepage/src/pages/admin/index.jsx [hunk 1]
```diff
@@ -3,10 +3,122 @@ import Head from 'next/head';
...
+          content={layout.get('text')?.toString()}
+          highlights={layout.get('highlights')?.toArray()}
+        />
+      );
+    } else if (layoutType === 'layout_headline') {
+      return (
+        <Headline
+          key={layout.get('title')?.toString()}
+          title={layout.get('title')?.toString()}
+          subtitle={layout.get('subtitle')?.toString()}
```
Comment (ljc1991) [inline homepage/src/pages/admin/index.jsx:44]: `<Headline>` component 好像沒有 `subtitle` 這個 property？

Comment (ben196888) [reply homepage/src/pages/admin/index.jsx:44]: 目前沒有，但是使用者有這個需求（許願要有這個需求），會在Headline裡增加subtitle

Comment (ben196888) [reply homepage/src/pages/admin/index.jsx:44]: 我會先移除，之後再開一個獨立的pull request來完成這個subtitle

Comment (ben196888) [reply homepage/src/pages/admin/index.jsx]: 已移除

**Pull request merged**